chainweb.cabal

```diff
@@ -1,7 +1,7 @@
 cabal-version: 2.2
 
 name:         chainweb
-version:      0.2
+version:      1.0
```

Comment:
Perhaps `1.0.0` to follow semver?

Reply:
This is Haskell.  We're using the PVP, not SemVer.